packages/tanstack-react-query/src/internals/createOptionsProxy.ts

```diff
@@ -172,32 +205,40 @@ export function createTRPCOptionsProxy<TRouter extends AnyRouter>(
   return createRecursiveProxy(({ args, path: _path }) => {
     const path = [..._path];
     const utilName = path.pop() as UtilsMethods;
-    const [input, userOptions] = args as any[];
+    const [arg1, arg2] = args as any[];
 
     const queryType = getQueryType(utilName);
-    const queryKey = getQueryKeyInternal(path, input, queryType);
+    const queryKey = getQueryKeyInternal(path, arg1, queryType ?? 'any');
 
     const contextMap: Record<UtilsMethods, () => unknown> = {
+      _input: null as any,
+      _output: null as any,
+      getQueryKey: () => queryKey,
+      invalidate: async () => {
+        await opts.queryClient.invalidateQueries({
+          queryKey: queryKey,
```

Comment:
again, queryKey here contains input but that's not what you have in the type above